Website/src/rendering/src/stories/Checkout/QuantityInput.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import QuantityInput from '../../components/Checkout/QuantityInput';
+import { RequiredDeep } from 'ordercloud-javascript-sdk';
+import { DPriceSchedule } from 'src/models/ordercloud/DPriceSchedule';
+
+export default {
+  title: 'Components/Checkout/QuantityInput',
+  component: QuantityInput,
+} as ComponentMeta<typeof QuantityInput>;
+
+const Template: ComponentStory<typeof QuantityInput> = (args) => <QuantityInput {...args} />;
+
+export const Default = Template.bind({});
```

Comment:
Please go ahead and add Redux and all required contexts to Storybook to be able to mock data for components using OrderCloud. This will be required for the build to pass anyway.

Once done, could we please duplicate this variant, name them RestrictedQuantity and UnrestrictedQuantity, and adjust the args to test each of them separately?